code/modules/projectiles/projectile/bullets.dm

```diff
@@ -1,8 +1,9 @@
 /obj/item/projectile/bullet
 	name = "bullet"
 	icon_state = "bullet"
-	damage = 60
+	damage = 45
 	damage_type = BRUTE
+	armor_penetration = 15

```

Comment:
Nope.
You added armor penetration to every bullet out there.
We have amor penetrating bullets for a reason.

Reply:
dang, I just noticed that